packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/examples/PeoplePicker.Types.Example.tsx

```diff
@@ -302,6 +301,15 @@ export class PeoplePickerTypesExample extends BaseComponent<any, IPeoplePickerEx
   }
 
   @autobind
+  private _onClickControlledPicker(item: IPersonaProps): () => void {
+    return (): void => {
```

Comment:
not sure why this needs to be a function returning.

Reply:
@dzearing Should I just revert this back to a lambda?